Exec/unit_tests/diffusion_test/analysis/profiles.py

```diff
@@ -22,6 +22,30 @@ def rgba_to_hex(rgba):
     b = int(rgba[2]*255.0)
     return '#{:02X}{:02X}{:02X}'.format(r, g, b)
 
+def get_analytic_profile(x_coord, time, problo, probhi,
```

Comment:
the analytic solution is a derived variable for this problem, so can't we just read it from the plotfile?

Reply:
yah, you're right. Didn't realize about that.